lib/shoulda/matchers/active_record/validate_uniqueness_of_matcher.rb

```diff
@@ -346,7 +346,8 @@ def existing_record_is_nil?
         end
 
         def existing_record_is_blank?
-          @existing_record.present? && existing_value.strip == ''
+          require "active_support/core_ext/object/blank"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.